src/braket/circuits/circuit.py

```diff
@@ -438,6 +440,136 @@ def add_circuit(
 
         return self
 
+    def add_noise(
+        self,
+        noise: Noise,
+        target_gates: Union[str, Iterable[str]] = None,
+        target_qubits: QubitSetInput = None,
+        target_times: Union[int, Iterable[int]] = None,
+        insert_strategy: str = "inclusive",
+    ) -> Circuit:
+        """
+        Add the provided `noise` to gates, qubits and time specified by `target_gates`,
+         `target_qubits` and `target_times`. When `target_gates`=None, `noise` is added
+        to the specified qubits and time. When `target_qubits`=None, `noise` is added to
+        the specified gates and time at all qubits. When `target_times`=None, `noise` is
+        added to the specified gates and qubits at all time. If all `target_gates`,
+        `target_qubits` and `target_times` are None, `noise` is added to every qubit at
+        every time in a circuit.
+
+        When `target_gates` is not None and `noise.qubit_count`>1, `noise.qubit_count`
```

Comment:
Format expressions in comment like you would in code: `noise.qubit_count` > 1

Reply:
Just to confirm, do you mean the spaces between "noise.qubit_count", ">", and  "1"?

Reply:
Yeah.